src/frontend/src/components/icons.ts

```diff
@@ -256,6 +256,21 @@ export const infoIcon = html`
   </svg>
 `;
 
+export const infoIconNaked = html`
```

Comment:
not sure what is meant by Naked would be super nice to have a comment or something here.